includes/class-deactivate.php

```
@@ -43,6 +43,21 @@ public static function init(): void {
 
 			Advanced_Cache_Handler::remove();
 
+			// Remove Redis object cache drop-in if it belongs to this plugin.
+			$object_cache_file = WP_CONTENT_DIR . '/object-cache.php';
```

PHPStan (2.2.5)

[warning] Constant WP_CONTENT_DIR not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)